index.md

```diff
@@ -176,12 +176,18 @@ situations.
 Container security separation policies are provided by a series of tools including SELinux, Capabilities, seccomp,
 and other security separation policies as specified in the OCI Specification.
 
-### Contribute
+## Contribute
+
+Join the CRI-O community with other users and contributors - whether you want to ask questions, report issues, submit pull requests, be a reviewer, help with documentation, improve our website, or write a blog post, we always welcome new members. If you contribute for the first time, check [How to contribute guide](https://github.com/cri-o/cri-o/blob/main/CONTRIBUTING.md).
```

Comment:
I would replace:
`If you contribute for the first time`
with
`For more guidance on how to contribute, please see the [contributing guide](...)`

Reply:
Got it and make sense. I will edit.